.github/workflows/linters.yml

```diff
@@ -45,18 +45,4 @@ jobs:
           npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
           [ -f .stylelintrc.json ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css-js/.stylelintrc.json
       - name: Stylelint Report
-        run: npx stylelint "**/*.{css,scss}"
```

Comment:
- Remember to avoid modifying the linters file. This project is the first one where you set up linters for JavaScript, HTML, and CSS and it will help to follow the best practices. Kindly add these lines back and solve any linters errors.